core/domain/activity_jobs_one_off_test.py

```diff
@@ -2137,3 +2137,291 @@ def test_missing_topic_rights_commit_logs(self):
         ]
 
         self.assertItemsEqual(expected_output, actual_output)
+
+
+class AddMissingCommitLogsJobTests(test_utils.GenericTestBase):
```

Comment:
Please also add tests for each of the cases to verify that the model was successfully added to the storage after the job runs (from what I can tell we currently are only verifying that the job reported success and that *an* entry was created, not that the entry is correctly created).

Reply:
I am not exactly sure what you mean by correctly created, should I be testing the attributes of the class also.

Reply:
That's what I mean, sorry: we should also verify the attributes of the newly created models to make sure we're correctly copying data over for those models.

Reply:
Done.